src/AppBundle/Controller/UserController.php

```diff
@@ -73,4 +74,24 @@ public function avatarAction($username, Request $request)
 
         return $response;
     }
+
+    /**
+     * @Route("/register/resend-email", name="register_resend_email", methods={"POST"})
+     * @Template()
+     */
+    public function resendRegistrationEmailAction(Request $request, UserManagerInterface $userManager, MailerInterface $mailer)
+    {
+        $emailExists = $request->request->has('email');
+        $userExists = $request->request->has('user');
+        if ($emailExists && $userExists == true) {
+            $email = $request->request->get('email');
+            $user = $userManager->findUserByEmail($email);
+            $mailer->sendConfirmationEmailMessage($user);
+        }
+
+        $this->session->set('fos_user_send_confirmation_email/email', $user->getEmail());
```

Comment:
So close... 😬
`$this->session` is `NULL` here. Check how to get the session in the [docs](https://symfony.com/doc/current/controller.html#managing-the-session)